src/utils/file-validators.ts

```diff
@@ -6,6 +6,12 @@ export interface HistoricData{
   value: number;
 }
 
+export interface PopulationData{
+  organizational_unit: string;
+  period: string;
+  value: number;  
+}
+
```

Comment:
Perhaps move this to a type folder or somewhere else more logical?

Reply:
Existing logic for other validators are in same file 
Then we need to do this for all validators.